core-concepts.md

```diff
@@ -6,47 +6,44 @@ number of client nodes, which are used for running jobs. You must also have a [C
 deployed (see the [Consul AWS Module](https://github.com/hashicorp/terraform-aws-consul)) in one of the following
 configurations:
 
-1. [Deploy Nomad and Consul in the same cluster](#deploy-nomad-and-consul-in-the-same-cluster)
-1. [Deploy Nomad and Consul in separate clusters](#deploy-nomad-and-consul-in-separate-clusters)
+  1. [Deploy Nomad and Consul in the same cluster](#deploy-nomad-and-consul-in-the-same-cluster)
+  2. [Deploy Nomad and Consul in separate clusters](#deploy-nomad-and-consul-in-separate-clusters)
 
 
 ## Deploy Nomad and Consul in the same cluster
 
 1. Use the [install-consul
-   module](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules/install-consul) from the Consul AWS
-   Module and the [install-nomad module](https://github.com/hashicorp/terraform-aws-nomad/tree/master/modules/install-nomad) from this Module in a Packer template to create
+   module](/modules/install-consul) from the Consul AWS
```

Comment:
The use of absolute URLs in our open source repos is intentional. This is a requirement for the URLs to work when rendered in the Terraform Registry.

Reply:
And if the IaC Lib code cannot handle absolute URLs right now, we'll need to fix that instead.

Reply:
It handles it. I just wanted to convert to relative URLs whenever I could to avoid issues where a link points to the master but not available in the latest release.